### PR TITLE
 Updated translations for new values added 

### PR DESCRIPTION
**Pull request by samforo**

### CODE REVIEW

File: brevent/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -174,18 +174,22 @@
     <string name="show_donation_rmb_and_contributor">Gracias por pagar %s Brefoil, y por su valiosa contribución a Brevent.</string>
     <string name="show_donation_play_and_rmb_and_contributor">Gracias por pagar %1$s Brefoil en Play Store, %2$s Brefoil, y por su valiosa contribución a Brevent.</string>
     <string name="brevent_network_adb">Red ADB</string>
+<<<<<<< HEAD
     <string name="brevent_language">Lenguaje</string>
+=======
+    <string name="brevent_language">Seleccionar lenguaje</string>
+>>>>>>> 7b4b31a5aa0e190fc465d2ec615ab618aaf646d9
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es/strings.xml]: It's a conflict, you should resolve it...

File: brevent/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -174,18 +174,22 @@
...
     <string name="brevent_about_version_summary_debug">%1$s, Edición de comunidad</string>
-    <string name="brevent_about_version_extra">\nStandby: %1$s\nStopped state：%2$s</string>
-    <string name="brevent_about_version_supported">supported</string>
-    <string name="brevent_about_version_unsupported">unsupported</string>
-    <string name="brevent_about_version_play">Play</string>
-    <string name="brevent_about_version_like_play">Como Play</string>
+    <string name="brevent_about_version_extra">\nStandby: %1$s\nForzar detener: %2$s</string>
+    <string name="brevent_about_version_supported">suportado</string>
+    <string name="brevent_about_version_unsupported">no soportado</string>
+    <string name="brevent_about_version_play">Play Store</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es/strings.xml:187]: Just keep original, short as possible.

It indicates the version. For spanish, normally, it's play version.
And if not installed from play store, it's like play version.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

File: brevent/src/main/res/values-es/strings.xml (hunk 2 above)
Comment (samforo) [reply brevent/src/main/res/values-es/strings.xml:187]: Play in Spanish has no coherent meaning as in English, can be understood as game. Play Store if you understand what it is. Hence my modification.

File: brevent/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (samforo) [reply brevent/src/main/res/values-es/strings.xml]: Ok

**Pull request merged**